app/views/questionnaire.py

```diff
@@ -273,19 +281,11 @@ def submit_answers(routing_path, eq_id, form_type, collection_id):
         if not sent:
             raise SubmissionFailedException()
 
-        current_app.eq['session_storage'].store_survey_completed_metadata(
-            metadata['tx_id'],
-            metadata['period_str'],
-            metadata['ru_ref'])
-
         get_questionnaire_store(current_user.user_id, current_user.user_ik).delete()
-        remove_survey_session_data()
 
-        return redirect(url_for(
-            '.get_thank_you',
-            eq_id=eq_id,
-            form_type=form_type,
-            collection_id=collection_id))
+        get_session_store().add_submission_time()
```

Comment:
do we want to be more explicit here and pass the time in?

Reply:
I didn't but go on then. Passing in datetime.utcnow().isoformat() to follow runner standards.